tests/migration/test_migrations.py

```
@@ -107,6 +117,31 @@ def test_migrate_verify_valid(monkeypatch, engine, tmp_path, only_last):
         raise e
 
 
+@pytest.mark.usefixtures("setup_test_schema")
+def test_migrate_verify_valid_skip(monkeypatch, engine, tmp_path):
```

Ruff (0.14.2)

Unused function argument: `monkeypatch`

(ARG001)